src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,398 @@
+		 * @param {Array} crumbs The array of all crumbs
+		 * @param {Array} newCrumbs The array of the crumbs to hide and add
+		 */
+		addCrumbs(crumbs, newCrumbs) {
+			newCrumbs.forEach((crumb, i) => {
+				if (crumb.elm && crumb.elm.classList) {
+					if (this.hiddenCrumbs.includes(crumb)) {
+						crumb.elm.classList.add('hidden')
+					} else {
+						crumb.elm.classList.remove('hidden')
+					}
+				}
+				crumbs.push(crumb)
+			})
+		},
+	},
+	render: function(createElement) {
+		// Get the breadcrumbs
+		const breadcrumbs = this.$slots.default || []
+
+		// Check that we have at least one breadcrumb
+		if (breadcrumbs.length === 0) {
+			return
+		}
+
+		// Add the root icon to the first breadcrumb
+		Vue.set(breadcrumbs[0].componentOptions.propsData, 'icon', this.rootIcon)
+
+		const crumbs = []
+		// The first half of the breadcrumbs
+		const crumbs1 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(0, Math.round(breadcrumbs.length / 2))
+			: breadcrumbs
+		this.addCrumbs(crumbs, crumbs1)
+
+		// The Actions menu
+		if (this.hiddenCrumbs.length) {
+			crumbs.push(createElement('div', { class: 'crumb' },
+				[createElement('Actions', {
+					class: 'dropdown',
+					props: {
+						forceMenu: true,
+					},
+					ref: 'actions',
+					nativeOn: {
+						dragstart: this.dragStart,
+						dragenter: () => { this.$refs.actions.opened = true },
+						dragleave: this.closeActions,
+					},
+				}, this.hiddenCrumbs.map(crumb => {
+					const to = crumb.componentOptions.propsData.to
+					const href = crumb.componentOptions.propsData.href
+					const disabled = crumb.componentOptions.propsData.disableDrop
+					let element = 'ActionLink'
+					let path = href
+					if (to) {
+						element = 'ActionRouter'
+						path = to
+					}
+					return createElement(element, {
+						class: 'crumb',
+						props: {
+							to: to,
+							href: href,
+							icon: 'icon-folder',
+						},
+						attrs: {
+							draggable: false,
+						},
+						nativeOn: {
+							dragstart: this.dragStart,
+							drop: ($event) => this.dropped($event, path, disabled),
+							dragover: this.dragOver,
+							dragenter: ($event) => this.dragEnter($event, disabled),
+							dragleave: ($event) => this.dragLeave($event, disabled),
+						},
+					},
+					crumb.componentOptions.propsData.title
+					)
+				}))]
+			))
+		}
+		// The second half of the breadcrumbs
+		const crumbs2 = this.hiddenCrumbs.length
+			? breadcrumbs.slice(Math.round(breadcrumbs.length / 2))
+			: []
+		this.addCrumbs(crumbs, crumbs2)
+
+		return createElement('div', { class: 'breadcrumb', ref: 'container' }, crumbs)
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+@import '../../fonts/scss/iconfont-vue';
+
+.breadcrumb {
+	width: 100%;
+	flex-grow: 1;
+
+	div.crumb {
+		@include iconfont('breadcrumb');
+		background-image: none;
+		display: inline-flex;
+		height: $clickable-area;
+		padding: 0;
+
+		&::before {
+			display: flex;
+			align-items: center;
+			order: 1;
+			color: #D7D7D7;
```

Comment:
Not compliant to the dark theme :)

Reply:
Hm, I just took the color from the breadcrumb.svg which is the same for light and dark. Which color should we use? `var(--color-main-text)`?

Reply:
I don't see any color variable which resembles the color we have at the moment.

Reply:
If at all `var(--color-text-maxcontrast)` comes close, but it doesn't seem to make much sense.

![Screenshot_2020-03-28 Inventar - Nextcloud](https://user-images.githubusercontent.com/2496460/77822671-23e91e00-70f5-11ea-9282-1a04dc1e6e35.png)
![Screenshot_2020-03-28 Inventar - Nextcloud(1)](https://user-images.githubusercontent.com/2496460/77822673-25b2e180-70f5-11ea-8b0e-8983d8bb1f3f.png)

Reply:
`var(--color-border-dark)` might work as well, but I think it is a bit to light in light theme and to dark in dark theme:

![Screenshot_2020-03-28 Inventar - Nextcloud(2)](https://user-images.githubusercontent.com/2496460/77822706-74f91200-70f5-11ea-8e1d-278112868471.png)
![Screenshot_2020-03-28 Inventar - Nextcloud](https://user-images.githubusercontent.com/2496460/77822705-73c7e500-70f5-11ea-8d3c-e3e41ec6fb54.png)

Reply:
Border seems more appropriate as this is after all, a border :stuck_out_tongue_closed_eyes:

Reply:
Fine with me, it will just change the current look a bit.